table/server/underdb/hive/src/main/java/alluxio/table/under/hive/util/HiveClientPool.java

```diff
@@ -41,24 +40,19 @@
 
   private final long mGcThresholdMs;
   private final String mConnectionUri;
-  private final String mHiveDbName;
-  /** This tracks if the db exists in HMS. */
-  private volatile boolean mDbExists = false;
 
   /**
    * Creates a new hive client client pool.
    *
    * @param connectionUri the connect uri for the hive metastore
-   * @param hiveDbName the db name in hive
    */
-  public HiveClientPool(String connectionUri, String hiveDbName) {
+  public HiveClientPool(String connectionUri) {
     super(Options.defaultOptions()
         .setMinCapacity(16)
         .setMaxCapacity(128)
```

Comment:
If we are re-using pools to the same hive db, we may need to change this to a configuration parameter?